src/pages/view-test-result/cat-c/__tests__/view-test-result.cat-c.page.spec.ts

```diff
@@ -0,0 +1,269 @@
+import { ComponentFixture, async, TestBed } from '@angular/core/testing';
+import {
+  IonicModule,
+  NavController,
+  NavParams,
+  Platform,
+  LoadingController,
+} from 'ionic-angular';
+import {
+  NavControllerMock,
+  NavParamsMock,
+  PlatformMock,
+  LoadingControllerMock,
+} from 'ionic-mocks';
+import { AppModule } from '../../../../app/app.module';
+import { AuthenticationProvider } from '../../../../providers/authentication/authentication';
+import { AuthenticationProviderMock } from '../../../../providers/authentication/__mocks__/authentication.mock';
+import { SearchProvider } from '../../../../providers/search/search';
+import { SearchProviderMock } from '../../../../providers/search/__mocks__/search.mock';
+import { MockComponent } from 'ng-mocks';
+import { TestDetailsCardComponent } from '../../components/test-details-card/test-details-card';
+import { RekeyDetailsCardComponent } from '../../components/rekey-details-card/rekey-details';
+import { RekeyReasonCardComponent } from '../../components/rekey-reason-card/rekey-reason';
+import { ExaminerDetailsCardComponent } from '../../components/examiner-details-card/examiner-details';
+import { By } from '@angular/platform-browser';
+import { ExaminerDetailsModel } from '../../components/examiner-details-card/examiner-details-card.model';
+import { TestDetailsModel } from '../../components/test-details-card/test-details-card.model';
+import { VehicleDetailsCardComponent } from '../components/vehicle-details-card/vehicle-details-card';
+import { categoryBETestResultMock } from '../../../../shared/mocks/cat-be-test-result.mock';
```

Comment:
Can we have a generic category mock or do we need to create a Cat C one?

Reply:
it needs to be created have added a TODO - we don't have the schema yet